src/content/docs/logs/get-started/live-archives.mdx

```diff
@@ -220,7 +197,7 @@ Once you query live archives, you can explore your compute consumption in the co
 Let's say you need to check if a user logged into a machine from their computer to fulfill a ticket on September 1st. You can use live archives to help confirm that this took place by completing the following steps:
 
 1. Check the time frame that the user logged into their machine.
-2. Test your query in <DoNotTranslate>**Logs**</DoNotTranslate> to ensure it's working as expected.
+2. Test your query in <DoNotTranslate>**Logs**</DoNotTranslate> to ensure it's working as expected. This is a dry run and won't return any data, but you're just validating the query. If it works, you can then run the more expensive query on your archived logs.
 
 Example query:
```

Comment:
@homelessbirds steps 3 and 4 are rendering as steps 1 and 2 in a separate ordered list. Similar to the issue above, you'd want to indent the Example Query phrase and the code snippet four spaces beyond step 2 (Test your query...)"